docs/conceptual/runtime/frame.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: Substrate Runtime Module Library
+title: FRAME
 ---
 
-The Substrate Runtime Module Library (SRML) is a collection of prebuilt runtime modules and support
-libraries which simplify development of a Substrate runtime.
+The __Framework for Runtime Aggregation of Modularised Entities (FRAME)__ is a set of modules (called pallets) and support libraries that simplify runtime development. Pallets are individual modules within FRAME that host domain-specific logic.
+
+FRAME provides some helper modules to interact with Substrate Primitives. Parity develops and maintains
```

Comment:
Let's give a quick (can be less than one sentence) description of what Substrate Primitives are. No detail, just don't want to lose people. Perhaps, like:
"...to interact with Substrate Primitives, which provide the interface to the core client."